joomla/joomla-cms
# [4.0] Capitalize "Hide all messages"

### Summary of Changes
For consistency, capitalize "Hide all messages" button in toolbar.


### Testing Instructions
Click Post Installation Messages button.
See capitalization in button of toolbar.
